app/models/efile_submission_dependent.rb

```diff
@@ -23,7 +23,7 @@ class EfileSubmissionDependent < ApplicationRecord
   belongs_to :efile_submission
   has_one :intake, through: :efile_submission
 
-  delegate :first_name, :last_name, :full_name, :ssn, :irs_relationship_enum, :birth_date, :tin_type_ssn?, :tin_type_atin?, :ip_pin, to: :dependent
+  delegate :first_name, :last_name, :full_name, :ssn, :irs_relationship_enum, :birth_date, :tin_type_ssn?, :tin_type_atin?, :ip_pin, :full_time_student_yes?, :permanently_totally_disabled_yes?, to: :dependent
```

Comment:
i had to add something to this list the other day too. maybe we should just delegate all undefined methods to dependent 🤔